gitpod-io/gitpod-packer-gcp-image
# Clean-up for context_user

## Description

If we don't do this, user is left dangling w/o a home directory, and is unable to SSH to corresponding nodes (unable to transfer keys).

## Related Issue(s)

n/a

## How to test

1. Confirm the logging tries to delete `kyle` twice. The first is hard coded, the second is contextual (using this code change).

```
    googlecompute.build_image: Removing files ...
    googlecompute.build_image: Removing user `kyle' ...
    googlecompute.build_image: Warning: group `kyle' has no more members.
    googlecompute.build_image: Done.
==> googlecompute.build_image: /usr/sbin/deluser: The user `kylos101' does not exist.
==> googlecompute.build_image: /usr/sbin/deluser: The user `kyle' does not exist.
```

2. SSH to a workspace-preview node...will it SSH?

## Release Notes

```release-note
NONE
```


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/gitpod-packer-gcp-image/pull/34"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/gitpod-packer-gcp-image/pull/34"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
